cmd/agent/common/common.go

```diff
@@ -44,9 +44,9 @@ var (
 func GetPythonPaths() []string {
 	return []string{
 		GetDistPath(),                                  // common modules are shipped in the dist path directly or under the "checks/" sub-dir
-		filepath.Join(GetDistPath(), "checks.d"),       // custom checks in the "checks.d/" sub-dir of the dist path
-		config.Datadog.GetString("additional_checksd"), // custom checks, have precedence over integrations-core checks
 		PySitePackages,                                 // integrations-core, wheels have precedence over old-school SDK
```

Comment:
So I think you're 100% right in your comment. This actually implies two things: 1) this was always broken as `site-packages` always took precedence over everything else since we append to `sys.path`. If you ask me the python stdlib and site-packages should probably always take precedence over any other code we ship, but maybe it's worth having that conversation.

Reply:
as you said I think it was "always broken". I don't have a strong opinion over whether we should first load the default python `sys.path` or our the code we ship. Happy to discuss it if you have arguments in favor of the first option :)